docs/docs/recipes/integrate-logto/android.mdx

```diff
@@ -1,17 +1,29 @@
 ---
-sidebar_label: Android (Kotlin)
+sidebar_label: Android
 ---
 
+import AppNote from './fragments/_app-note.mdx';
+import ConfigureRedirectUri from './fragments/_configure-redirect-uri.mdx';
+import SignInNote from './fragments/_sign-in-note.mdx';
+import FurtherReadings from './fragments/_further-readings.md';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import redirectUriFigure from './assets/android-redirect-uri.png';
 
-# Android: Integrate `io.logto.sdk:android`
+
+# Android: Integrate Logto Android SDK
```

Comment:
Fix these lint issues and we are good to move on

Reply:
fixed.